### ISSUE TITLE
Add the CNC room and vinyl-cutter/sublimation-printing area to the booking system

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#1469]
On the Google Group, @DuncReid suggested:
> Would it be an idea – and I know this would cause work 😊 – to add the CNC & Sublimation spaces into the booking system? Often it’s the sort of thing that people want to come in and spend an hour or two doing a batch of stuff so if they journey in only to find it in use it’s going to get a bit awkward.

It would make sense.  Ideally we'd use [our new booking system](https://github.com/DoESLiverpool/optimism), but that's not ready yet.  As an interim option we can probably use the same service that we do for the laser-cutters and desks.

**Comment by amcewen** [comment]
I've made a start on this.  There are two new booking pages in our youcanbook.me account: one for the CNC room, and one for the workshop recess (vinyl cutting and sublimation printer area).  I (think I) have updated the notifications, etc. to refer to things correctly (I duplicated the desk booking, as that seemed closest).

I've also created new resources in our Google Apps account to create new calendars to use for it.  I think I've given the youcanbook.me account access to them, but they're not showing up when I try to change the calendar it's using :-/

I'm going to leave it for a bit to see if the list of calendars is being cached somewhere.  Then we need to double-check you can't do multiple bookings at the same time (I couldn't spot where that's set, but maybe it's in the calendar section that I've not updated yet).

**Comment by helicalbytes** [comment]
Is there a plan to look at development of the new booking system at the upcoming hack the space meeting on 4th August? I would be interested in being involved in it, if that's possible.

**Comment by amcewen** [comment]
@helicalbytes Yes!  That is the plan (and it'd be great if you can join in!)  I guess we'll gather on https://meet.jit.si/bubblino at ~6:30pm and work things out from there.

**Comment by amcewen** [comment]
This is being covered by [optimism](https://github.com/DoESLiverpool/optimism/) at the hack-the-space evenings, and the plan is to soft-launch that to do bookings for the CNC and embroidery machine (and once we're happy with it we can extend it to the lasers and desks...)

**Comment by johnmckerrell** [comment]
I just spotted that this is now a feature in Google Calendar. There's something about the appointment being added to your personal default calendar as well as the custom calendar which I think is the thing that will make it unworkable for us but probably worth someone investigating this more. https://lifehacker.com/you-can-finally-offer-bookable-appointments-on-your-goo-1850853638
